woocommerce/woocommerce-android
# Remove wcpayandstripeinstalled state from the codebase

Closes: #6884 


### Description

Now that we are allowing the merchant to select their preferred plugin when multiple plugins are found on their site. We can remove the WcPayAndStripeInstalledState state from the codebase which isn't used anywhere and is already being replaced by SelectPaymentPluginState

This PR also removes IPP gateway selection feature flag from the codebase.

### Testing instructions

1. Ensure that the CI is green
2. Ensure happy flows of the onboarding screen works as expected.
